src/frontend/download-chrome.ts

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env node
+import * as os from "os";
```

Comment:
even though it is not that super important since not bundled, I would avoid importing `*` anyway as a best practice

Reply:
agreed; copied over from original script. Will change